app/src/main/java/com/navapp/navigation/route/Route.java

```diff
@@ -1,39 +1,27 @@
 package com.navapp.navigation.route;
 
 import com.navapp.navigation.destination.Destination;
-import com.navapp.navigation.destination.data.Address;
 
-import java.util.ArrayDeque;
-import java.util.ArrayList;
-import java.util.Collection;
 import java.util.Collections;
-import java.util.Deque;
 import java.util.List;
 
 public class Route {
     private final List<Destination> destinations;
-    private final Address end;
 
-    public Route(Collection<Destination> destinations, Address end) {
-        this.destinations = new ArrayList<>(destinations);
-        this.end = end;
-    }
-
-    public Route changeEnd(Address newEnd) {
-        return new Route(destinations, newEnd);
-    }
-
-    public Route addDestination(Destination destination) {
-        List<Destination> newList = new ArrayList<>(destinations);
-        newList.add(destination);
-        return new Route(newList, end);
+    public Route(List<Destination> destinations) {
+        this.destinations = destinations;
     }
 
     public List<Destination> getDestinations() {
         return Collections.unmodifiableList(destinations);
     }
 
-    public Address getEnd() {
-        return end;
+    public double getSum() {
```

Comment:
Sum of what?

Reply:
Some of the rates, I thought about changing it to it, but figured that it's easy to figure out

Reply:
renamed